### PR TITLE
db-front end hook up

### PR DESCRIPTION
**Pull request by bbrower21**
-Added new filter logic for db queries for compatibility with better-sqlite3
-Cleaned up naming conventions on front-end 
-Connected filter.ts file to main.dev.ts

### CODE REVIEW

File: app/components/LogsView.tsx [hunk 1]
```diff
@@ -39,31 +39,34 @@ const LogsView = ({ filterOptions }) => {
 
         if (date < from || date > to) return null;
       }
-
-      if (currentOption.length && !currentOption.includes(logEntry[option]))
+      console.log('log entry:', logEntry);
```
Comment (noodlemonkey) [inline app/components/LogsView.tsx:42]: do we need this console log

**Review by samuy — APPROVED** [review]

File: app/components/LogsView.tsx [hunk 2]
```diff
@@ -39,31 +39,34 @@ const LogsView = ({ filterOptions }) => {
...
+      // console.log('logEntry[option]:', logEntry)
+      if (currentOption.length && !currentOption.includes(logEntry[option])){
+      // console.log('returning null');
```
Comment (noodlemonkey) [inline app/components/LogsView.tsx:45]: the ghost of console logs past

File: app/components/LogsView.tsx [hunk 3]
```diff
@@ -111,12 +114,14 @@ const LogsView = ({ filterOptions }) => {
...
   // Filter logic
   useEffect(() => {
+    console.log('useEffect in filterOptions ipcRendereer effect')
     ipcRenderer.send('filter', filterOptions);
   }, [filterOptions]);
 
   useEffect(() => {
     ipcRenderer.on('reply-filter', (event, arg) => {
       setLogs(arg);
+      console.log('arg on front-end: ', arg)
```
Comment (noodlemonkey) [inline app/components/LogsView.tsx:124]: console logs

**Review by noodlemonkey — APPROVED** [review]

File: app/components/LogsView.tsx (hunk 2 above)
Comment (samuy) [reply app/components/LogsView.tsx:45]: 👻

**Pull request merged**